python/paddle/fluid/param_attr.py

```diff
@@ -180,14 +180,14 @@ def _to_kwargs(self, with_initializer=False):
 class WeightNormParamAttr(ParamAttr):
     """
     Used for weight Norm. Weight Norm is a reparameterization of the weight vectors
-    in a neural network that decouples the length of those weight vectors from
+    in a neural network that decouples the magnitude of those weight vectors from
     their direction. Weight Norm has been implemented as discussed in this
     paper: `Weight Normalization: A Simple Reparameterization to Accelerate
     Training of Deep Neural Networks
     <https://arxiv.org/pdf/1602.07868.pdf>`_.
 
     Args:
-        dim(list): The parameter's name. Default None.
+        dim(int): Dimension over which to compute the norm. Default None.
```

Comment:
Might it be the  Dimension except which

Reply:
this is referred to pytorch, which said 'dim (int, optional): dimension over which to compute the norm
', just keep consistent with other framework